src/puppetlabs/trapperkeeper/services/protocols/metrics.clj

```diff
@@ -2,5 +2,9 @@
 
 (defprotocol MetricsService
   "A service that tracks runtime metrics for the process."
-  (get-metrics-registry [this] "Provides access to the MetricsRegistry instance
-                                where metrics are stored."))
\ No newline at end of file
+  (get-metrics-registry
+    [this]
+    [this registry opts]
+    "Provides access to a MetricsRegistry configured with ops where `registry`
+     is the key used to look up the registry. Specifing no `registry` will
+     return the default MetricsRegistry"))
```

Comment:
What is the type for `registry`?  Docs on what `opts` can include?  A reference to a prismatic schema would be ideal.